CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 0.2.1 (WIP) ##
 
-This version is based on [blobfuse2 2.1.1](https://github.com/Azure/azure-storage-fuse/releases/tag/blobfuse2-2.1.1) (upstream).
+This version is based on [blobfuse2 commit 9c10ae08 dated 2023-11-7](https://github.com/Azure/azure-storage-fuse/commit/9c10ae0880e4b042d89ae4ea571aaf8f49b3d519) (upstream).
 
```

Comment:
Could you add that this changes sync-to-flush to default to true.

Reply:
Good point!